### ISSUE TITLE
path error in js

### ISSUE THREAD

**Issue by Lemez** [ramanarunachalam/Music#1]
Hello
Wonderful site for fans of Carnatic music, thank you. But please note that the js path for the carnatic player gives a 404, so the player is not working.

**Comment by ctatachar** [comment]
Yes, I noticed the same and I didn't see the Audios folder and audio assets referred to in the onclick , so perhaps it's still a WIP? A really great effort organizing these nevertheless... thanks much! It would be incredible to have the mp3s available!! Esp. SKR's as they're so rare and hard to find. Would be glad to assist if any help is needed :)
